### ISSUE TITLE
react-native-navigation completely breaks react native project

### ISSUE THREAD

**Issue by bootrino** [wix/react-native-navigation#6380]
It's posted over at the base react native issues, but at this stage the cause of the problem, at least in my case, is react-native-navigation:

https://github.com/facebook/react-native/issues/28408#issuecomment-656356786

I can't find the way to recover my project after installing react-native-navigation and have to rebuild it step by step from scratch.

**Comment by jinshin1013** [comment]
HI @bootrino, reading the commands you have run in that comment, it seems like you haven't run `react-native link react-native-navigation` which is required as RNN requires native code changes. 

Would you be able to provide a minimal reproduction on Github for me to debug? Cheers!

**Comment by bootrino** [comment]
None of the proposed solutions worked for me.

Finally I think I have fixed it:

I got a new hard disk for my Mac
I installed a totally clean build of the very latest OSX Catalina (NOT an upgrade)
I reinstalled all of my applications
I did a clean install of  xcode
I did a clean install of Android Studio

Goodness knows what was wrong with my toolchain but a lovely clean build of the latest everything seems to work just fine.

Radical action that's for sure but what's the point in trying to diagnose weird errors on an out of date system.
